### PR TITLE
Allow for longer subscription titles and handle title overflow in subscription card

### PR DESCRIPTION
**Pull request by ashwin003**
Addresses #92 

- Increases title max length to 20
- Adds text truncation for titles in the main and monthly pages

### CODE REVIEW

File: subtrack.DAL/Entities/Subscription.cs [hunk 1]
```diff
@@ -7,7 +7,7 @@ public class Subscription
     public int Id { get; set; }
 
     [Required]
-    [StringLength(12,ErrorMessage = "Service name should be less than 12 characters.")]
+    [StringLength(20,ErrorMessage = "Service name should be less than 20 characters.")]
```
Comment (chrisK00) [inline subtrack.DAL/Entities/Subscription.cs]: What was the max title length on sub details?

Comment (ashwin003) [reply subtrack.DAL/Entities/Subscription.cs]: subscription details page? I think that just splits the title across multiple lines if it's too long.

Comment (chrisK00) [reply subtrack.DAL/Entities/Subscription.cs]: Then we can set it to something like 50. You could probs just do the edit here on github directly, 50 here and 50 on the error message

**Review by chrisK00 — APPROVED** [review]
🔥🔥🔥🔥

**Pull request merged**